gson/src/main/java/com/google/gson/internal/bind/DateTimeTypeAdapters.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2011-2016 Google Inc.
```

Comment:
Off-topic, since Google is no longer involved in any way with Gson, what do we do?